components/brave_wallet_ui/panel/BUILD.gn

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
we probably could put wallet webuis into `components/brave_wallet/webui`

Reply:
Posted https://github.com/brave/brave-browser/issues/16051 to track this suggestion